docs/Using-Fleet/REST-API.md

```diff
@@ -5480,21 +5504,28 @@ load balancer timeout.
   ]
 }
 ```
+
 ---
 
 ## Schedule
 
-- [Get schedule](#get-schedule)
-- [Add query to schedule](#add-query-to-schedule)
-- [Edit query in schedule](#edit-query-in-schedule)
-- [Remove query from schedule](#remove-query-from-schedule)
+> The Schedule API endpoints are deprecated as of Fleet 4.XX. It is maintained for backwards compatibility. 
+> Please use the [Queries](#queries) endpoint, which as of 4.xx has attributes such as `interval` and `platform` that enable scheduling.
```

Comment:
`4.xx` here but `4.XX` above.

Reply:
.xx vs. .XX will be updated to the proper release number when we decide what release this is in